lib/css/modules/_showImages.scss

```diff
@@ -317,6 +319,23 @@ div.madeVisible {
 			padding: 0 1%;
 			box-sizing: border-box;
 
+			.video-advanced-position-cursor {
+				display: none;
+				position: relative;
+				top: -9px;
+				height: 16px;
+				width: 16px;
+				margin-left: calc(-1% + -9px);
```

Comment:
This `calc` is a bit strange, it seems to work fine with `margin-left: -12px;`. Is there something I'm missing?